frontend/src/store/authn/helper.js

```diff
@@ -190,6 +204,8 @@ export function useUserManager (options) {
       let frameRequestCallback
       if (isNoUserError(err)) {
         frameRequestCallback = () => signin()
+      } else if (isSessionExpiredError(err)) {
+        frameRequestCallback = () => signout()
```

Comment:
usually this should not happen as we have the expiration timer here, right?https://github.com/gardener/dashboard/blob/2a611f651de356816a85d956a7ef516594c7031b/frontend/src/router/guards.js#L57


In the router guard we pass the `redirectPath` to `signout` when the session expires
https://github.com/gardener/dashboard/blob/2a611f651de356816a85d956a7ef516594c7031b/frontend/src/router/guards.js#L58-L59

but in this case, when the session expires, we don't. 
```js
      } else if (isSessionExpiredError(err)) {
        frameRequestCallback = () => signout()
```

Also, shouldn't we, before we call `signout` or `signin` also log the reason? like we do it [here](https://github.com/gardener/dashboard/blob/2a611f651de356816a85d956a7ef516594c7031b/frontend/src/router/guards.js#L58)

Reply:
the error is logged a few lines above.. closing

Reply:
The error is logged in all cases in the first line of the try catch block. The session expired error should not occur if everthing works as expected. If it happens anyway, it is acceptable to lose the context.